Ghpr.CouchDb/Entities/RunSummary.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Newtonsoft.Json;
+
+namespace Ghpr.CouchDb.Entities
+{
+    [JsonObject(MemberSerialization.OptIn)]
+    public class RunSummary
+    {
+        [JsonProperty(PropertyName = "total")]
+        public int Total { get; set; }
+
+        [JsonProperty(PropertyName = "success")]
+        public int Success { get; set; }
+
+        [JsonProperty(PropertyName = "errors")]
+        public int Errors { get; set; }
+
+        [JsonProperty(PropertyName = "failures")]
+        public int Failures { get; set; }
+
+        [JsonProperty(PropertyName = "inconclusive")]
+        public int Inconclusive { get; set; }
+
+        [JsonProperty(PropertyName = "ignored")]
+        public int Ignored { get; set; }
+
+        [JsonProperty(PropertyName = "unknown")]
+        public int Unknown { get; set; }
+
+        public RunSummary(int total = 0, int success = 0, int errors = 0, int failures = 0, int inconclusive = 0, int ignored = 0, int unknown = 0)
```

Comment:
All integers as valuetapys have a default value of 0 already. No need for this constructor